### PR TITLE
Bug 1733207: Make unpublishing release notes more obvious

### PR DESCRIPTION
**Pull request by pascalchevrel**
1/ Adds a new option to the admin listing to unpublish releases
2/ Put the is_public checkbox at the top with a note explaining what it does

### CODE REVIEW

File: nucleus/rna/models.py [hunk 1]
```diff
@@ -35,14 +35,14 @@ class Release(SaveToGithubModel):
                 'Firefox Extended Support Release', 'Firefox OS',
                 'Thunderbird', 'Firefox for iOS')
 
+    is_public = models.BooleanField(default=False, help_text = "Note: If checked, these Release Notes will be visible on www.mozilla.org")
```
Comment (pmac) [inline nucleus/rna/models.py:38]: Unfortunately the order here doesn't affect the order on the admin page. We need to define the list of fields in the [`fields` param of the admin class](https://docs.djangoproject.com/en/2.2/ref/contrib/admin/#django.contrib.admin.ModelAdmin.fields).

Comment (pmac) [reply nucleus/rna/models.py:38]: I should say I don't think it reliably affects the order, the way to make sure is in the docs I liked above.

Comment (pascalchevrel) [reply nucleus/rna/models.py:38]: I tried locally and it works for me:
image

**Comment by pascalchevrel** [comment]
I implemented the requested changes and rebased the branch, does it look correct now? Thanks

File: nucleus/rna/models.py (hunk 1 above)
Comment (pmac) [reply nucleus/rna/models.py]: huh. cool. well maybe that's fine then.

**Comment by pmac** [comment]
Thanks Pascal! Really appreciate your help with all of this. Do you think we should change the wording of the option to something more like "Published"? We could have the actions be "Publish" and "Unpublish"?

**Comment by pmac** [comment]
There's also the option to have check boxes for the is_public option on the list page itself so you can change multiple of them at a time by just selecting those and hitting "Save" instead of using the actions drop down, since it's just a single field change.

**Pull request merged**